faststreams/outputs.nim

```diff
@@ -795,9 +826,28 @@ template getOutput*(s: OutputStream, T: type seq[byte]): seq[byte] =
 template getOutput*(s: OutputStream): seq[byte] =
   cast[seq[byte]](s.getOutput(string))
 
-template getOutput*(s: AsyncOutputStream): seq[byte] =
-  getOutput OutputStream(s)
+when fsAsyncSupport:
+  template getOutput*(s: AsyncOutputStream): seq[byte] =
+    getOutput OutputStream(s)
+
+  template getOutput*(s: AsyncOutputStream, T: type): untyped =
+    getOutput OutputStream(s), T
+
+  when fsAsyncSupport:
```

Comment:
Why this double `when fsAsyncSupport` nesting here?

Reply:
oops, probably a leftover from trying to not repeat the whole block